### PR TITLE
[Homepage prep] Add events slideshow to homepage

### PR DESCRIPTION
**Pull request by davidrapson**
Extracts the new homepage event panels from https://github.com/guardian/membership-frontend/pull/700.

**New**
screen shot 2015-09-02 at 17 47 39

**Old**
screen shot 2015-09-02 at 17 27 25

events mov

This leaves https://github.com/guardian/membership-frontend/pull/700 as only containing updates to the benefits table.

@chrisjowen

### CODE REVIEW

File: frontend/app/services/EventbriteService.scala [hunk 1]
```diff
@@ -77,6 +79,7 @@ trait EventbriteService extends WebServiceHelper[EBObject, EBError] {
   def getEventsByIds(ids: Seq[String]): Seq[RichEvent] = events.filter(e => ids.contains(e.event.id))
   def getLimitedAvailability: Seq[RichEvent] = events.filter(_.event.isLimitedAvailability)
   def getRecentlyCreated(start: DateTime): Seq[RichEvent] = events.filter(_.created.isAfter(start))
+  def getSortedByCreationDate: Seq[RichEvent] = events.sortBy(_.created.toDateTime).reverse
```
Comment (afiore) [inline frontend/app/services/EventbriteService.scala]: This is suboptimal as it involves two traversals of the list. Alternatively, you can supply a reverse ordering and do it in one traversal only
```scala
events.sorted(Ordering[DateTime].reverse)
```

File: frontend/app/views/fragments/media/slideshowEvents.scala.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
+@(events: Seq[model.RichEvent.RichEvent])
+
+<div class="slideshow slideshow--events js-slideshow">
+    <div class="slideshow__container">
+        @for(event <- events if event.imgOpt.nonEmpty) {
```
Comment (afiore) [inline frontend/app/views/fragments/media/slideshowEvents.scala.html]: Are we omitting here any events that have no image?

Comment (davidrapson) [inline frontend/app/views/fragments/media/slideshowEvents.scala.html]: Yep, that's the intention. Let me know if there's a better way.

File: frontend/app/services/EventbriteService.scala (hunk 1 above)
Comment (davidrapson) [inline frontend/app/services/EventbriteService.scala]: Fantastic, thanks.

File: frontend/app/views/fragments/media/slideshowEvents.scala.html (hunk 2 above)
Comment (afiore) [inline frontend/app/views/fragments/media/slideshowEvents.scala.html]: I think is ok, just wanted to check on the intent...

**Comment by afiore** [comment]
:+1:

**Pull request merged**